.drone.star

```diff
@@ -133,6 +134,25 @@ config = {
                 "POSTPROCESSING_DELAY": "30s",
             },
         },
+        "apiEmailNotification": {
+            "suites": [
+                "apiEmailNotification",
+            ],
+            "skip": False,
+            "earlyFail": True,
+            "emailNeeded": True,
+            "extraEnvironment": {
+                "EMAIL_HOST": "email",
+                "EMAIL_PORT": "9000",
+            },
+            "extraServerEnvironment": {
+                "NOTIFICATIONS_SMTP_HOST": "email",
+                "NOTIFICATIONS_SMTP_PORT": "2500",
+                "NOTIFICATIONS_SMTP_SENDER": "ownCloud <noreply@owncloud.com>",
+                "NOTIFICATIONS_SMTP_USERNAME": "ownCloud <noreply@owncloud.com>",
```

Comment:
Are these required? maybe there is default?

Reply:
the default sender on config is `ownCloud <noreply@example.com>` . Should we keep the default or use `owncloud.com` domain. If we use the default then we can remove these two environment variables.